playbooks/roles/edx_django_service/tasks/main.yml

```diff
@@ -184,7 +187,7 @@
   pip:
     name: "{{ item.name }}"
     version: "{{ item.version|default(omit) }}"
-    extra_args: "--exists-action w {{ item.extra_args|default('') }}"
+    extra_args: "--no-deps --exists-action w {{ item.extra_args|default('') }}"
```

Comment:
`--no-deps` may break de installation of any package that is added to `{{ <DJANGO_SERVICE>_EXTRA_REQUIREMENTS }}`